### PR TITLE
POWRNG-40

### PR DESCRIPTION
**Pull request by RafisSomeone**

### CODE REVIEW

File: src/app/components/login-page/login-page.component.ts [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+  selector: 'app-login-page',
+  templateUrl: './login-page.component.html',
+  styleUrls: ['./login-page.component.css']
+})
+export class LoginPageComponent implements OnInit {
+  roleControl = new FormControl('', Validators.required);
+  public disabled: boolean
+  public roleSelected;
+  user: User;
+  private readonly URL = 'http://localhost:8099/api/users';
```
Comment (Sroczek) [inline src/app/components/login-page/login-page.component.ts:17]: It is good practice to store variables like url in enviroment.ts and environment.prod.ts

File: src/app/components/header/header.component.ts [hunk 2]
```diff
@@ -7,7 +8,9 @@ import { Component, OnInit } from '@angular/core';
 })
 export class HeaderComponent implements OnInit {
 
-  constructor() { }
+  constructor(private router:Router) { 
+    console.log(this.router.url)
```
Comment (Sroczek) [inline src/app/components/header/header.component.ts:12]: Is this an intentional log?

File: src/app/components/login-page/login-page.component.ts (hunk 1 above)
Comment (RafisSomeone) [reply src/app/components/login-page/login-page.component.ts:17]: Done.

File: src/app/components/header/header.component.ts (hunk 2 above)
Comment (RafisSomeone) [reply src/app/components/header/header.component.ts:12]: Cleared.

**Pull request merged**